migrations/manual/educandu-2021-12-13-01-make-cdn-urls-generic.js

```diff
@@ -0,0 +1,55 @@
+async function updateAll(collection, query, updateFn) {
+  const cursor = collection.find(query);
+
+  /* eslint-disable-next-line no-await-in-loop */
+  while (await cursor.hasNext()) {
+    /* eslint-disable-next-line no-await-in-loop */
+    const doc = await cursor.next();
+    /* eslint-disable-next-line no-await-in-loop */
+    const updatedDoc = await updateFn(doc);
+    if (updatedDoc) {
+      /* eslint-disable-next-line no-await-in-loop */
+      await collection.replaceOne({ _id: doc._id }, updatedDoc);
+    }
+  }
+}
+
+export function processSection(section) {
+  let updateCount = 0;
+  if (section.type === 'markdown' && section.content) {
+    const regexp = /\]\((https:\/\/((staging\.)?cdn\.elmu\.online)|(https:\/\/cdn\.((integration|staging)\.)?openmusic\.academy))\//g;
```

Comment:
Not I noticed, I think here the parenthesis do not close correctly 🤔 
Should be `online))|(`, an extra closed parenthesis there before the pipe

Reply:
Never mind, wrong escaped parenthesis count from my side